src/functions-and-arrays.js

```diff
@@ -1,19 +1,116 @@
 // Iteration #1: Find the maximum
+function maxOfTwoNumbers(num1, num2) {
```

Comment:
What does this function return if the numbers are the same?
Perhaps Google "max of two numbers in JavaScript" to see different ways of solving this one. :)